README.md

```diff
@@ -238,6 +238,12 @@ The filtering can also be applied for pins with `subscribe` attribute.
 
 ## Changelog
 
+### v4.7.2
+
+#### Fixed:
+
+* Codec now publishes an error event and an error message successfully when it cannot parse a th2-message
```

Comment:
was it unsuccessfully before? Please add some description about why, just a few words